packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -414,10 +415,11 @@ export function getInternalReactConstants(
       case IndeterminateComponent:
         return getDisplayName(resolvedType);
       case ForwardRef:
```

Comment:
This function has a notice regarding syncing ("// NOTICE Keep in sync with shouldFilterFiber() and other get*ForFiber methods") but I guess this only applies to what fiber types are handled not to how the display name is computed?